build.gradle

```diff
@@ -18,6 +18,7 @@ targetCompatibility = 1.6
 repositories {
     maven { url "http://oss.sonatype.org/content/repositories/snapshots" }
     maven { url "http://repo.maven.apache.org/maven2" }
+    mavenLocal();
```

Comment:
No reason that I am aware of, if it doesn't get included here I'll just update the gradle later.